vorestation.dme

```diff
@@ -2401,6 +2401,11 @@
 #include "code\modules\vore\fluffstuff\custom_items_vr.dm"
 #include "code\modules\vore\fluffstuff\custom_mecha_vr.dm"
 #include "code\modules\vore\fluffstuff\custom_permits_vr.dm"
+#include "code\modules\vore\fluffstuff\guns\bsharpoon.dm"
+#include "code\modules\vore\fluffstuff\guns\dominator.dm"
+#include "code\modules\vore\fluffstuff\guns\protector.dm"
+#include "code\modules\vore\fluffstuff\guns\pummeler.dm"
+#include "code\modules\vore\fluffstuff\guns\sickshot.dm"

```

Comment:
This refactor of how we handle uniquely coded guns is probably how we should do this in future. However I would move these under the original /guns directory, and add a /vore or /vorestation folder to put these in.